CDP4JsonFileDal/JsonFileDalUtils.cs

```diff
@@ -168,30 +168,20 @@ internal static void AddDomainsOfExpertise(EngineeringModelSetup engineeringMode
         /// </param>
         /// <param name="organizations">
         /// The target <see cref="HashSet{Organization}"/>
-        /// </param>///
-        internal static void AddPersons(EngineeringModelSetup engineeringModelSetup, ref HashSet<Person> persons, ref HashSet<PersonRole> personRoles, ref HashSet<ParticipantRole> participantRoles, ref HashSet<Organization> organizations)
+        /// </param>
+        internal static void AddPersons(
+            EngineeringModelSetup engineeringModelSetup,
+            ref HashSet<Person> persons,
+            ref HashSet<PersonRole> personRoles,
+            ref HashSet<ParticipantRole> participantRoles,
+            ref HashSet<Organization> organizations)
         {
             foreach (var participant in engineeringModelSetup.Participant)
             {
-                if (!persons.Contains(participant.Person))
-                {
-                    persons.Add(participant.Person);
-                }
-
-                if (!personRoles.Contains(participant.Person.Role))
-                {
-                    personRoles.Add(participant.Person.Role);
-                }
-
-                if (!participantRoles.Contains(participant.Role))
-                {
-                    participantRoles.Add(participant.Role);
-                }
-
-                if (!organizations.Contains(participant.Person.Organization))
-                {
-                    organizations.Add(participant.Person.Organization);
-                }
+                persons.Add(participant.Person);
```

Comment:
Just to be clear, `Person`, `PersonRole`, `ParticipantRole`, and `Organization` **DO** implement the `Equals` and `GetHashCode` methods?